### ISSUE TITLE
[REQ] Grid - Grouping with Summary

### ISSUE THREAD

**Issue by amanu** [vitmalina/w2ui#60]
many business use cases require in grids grouping of the data, but also summaries using various functions. see in action:

http://docs.sencha.com/ext-js/4-2/extjs-build/examples/grid/group-summary-grid.html

of course, sometimes the summary calculations needs to be done on the server to have the correct value, since not always the entire information is available in the client to do the calculations. see in action:

http://docs.sencha.com/ext-js/4-2/extjs-build/examples/grid/remote-group-summary-grid.html

regards

**Comment by prophe05** [comment]
+1

It will be very helpful to have possibility to create grid like this http://mleibman.github.io/SlickGrid/examples/example-grouping

**Comment by kstan79** [comment]
yes, prophe05 gave good example, grouping with total is very useful function since user like to manipulate data in run time
